### ISSUE TITLE
White "turn direction" arrows on the blue path are not visible at the daylight in navigation mode

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#1086]
It's a pain for cyclists and city walkers.

![2021-08-12 20 20 58](https://user-images.githubusercontent.com/170263/129248563-ee1ccf68-dfe2-49af-bc90-ca8ab87b6de1.jpg)

**Comment by biodranik** [comment]
If an intermediate stop is added, some arrows already change their color.

**Comment by biodranik** [comment]
@strump can you please test similar routes with the route drawn on top of buildings?

**Comment by strump** [comment]
I patched `front_enderer.cpp:1434` changing order or rendering 3D buildings. This is how bicycle route looks now.

| Google Play  | Route ot top |
| ------------- | ------------- |
| ![2023-04-25 21 46 00](https://user-images.githubusercontent.com/720808/234377369-c4cb5f40-fd89-466f-9fb3-833ec32faf0c.jpg) | ![2023-04-25 21 45 32](https://user-images.githubusercontent.com/720808/234377423-53f16ca4-0752-4e14-b1d0-5cd632f72cde.jpg) |
| ![2023-04-25 21 52 15](https://user-images.githubusercontent.com/720808/234377534-2be1d436-cdfb-464d-9254-50877be04ced.jpg) | ![2023-04-25 21 52 38](https://user-images.githubusercontent.com/720808/234377579-c904b807-7a4d-4ebc-ab57-5ce9f5e2b60b.jpg) |

**Comment by biodranik** [comment]
It looks ok to me. Let's test this branch (always on top of buildings) and if nobody complains then we can merge it.

**Comment by kuba-orlik** [comment]
It does increase readability a bit, however I think the biggest culprit here is the fact that the turn arrows are white instead of purple (or whichever color the dashed line is at the moment)

**Comment by louwers** [comment]
> biggest culprit here is the fact that the turn arrows are white instead of purple

Yep... This makes it very hard to see the route. Sometimes the whole route looks like it is gone.
